# Description: use the same .egg string format than Debian python-setuptools
# Author: Carl Chenet <chaica@ohmytux.com>

--- a/pip/req.py
+++ b/pip/req.py
@@ -420,7 +420,7 @@
         paths_to_remove = UninstallPathSet(dist)
 
         pip_egg_info_path = os.path.join(dist.location,
-                                         dist.egg_name()) + '.egg-info'
+                                         dist.egg_name().split('-py')[0]) + '.egg-info'
         # workaround for http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=618367
         debian_egg_info_path = pip_egg_info_path.replace(
             '-py%s' % pkg_resources.PY_MAJOR, '')
